templates/hello-world/contribution.ts

```diff
@@ -8,7 +8,7 @@ export const <%= params.extensionPrefix %>Command = {
 };
 
 @injectable()
-export class <%= params.extensionPrefix %>CommandContribution implements CommandContribution {
+export class <%= params.extensionPrefix %>CommandContribution implements CommandContribution
```

Comment:
I think there is a mistake in this line :-)

Reply:
Ahh you're right ;)